src/rpc/net.cpp

```diff
@@ -110,6 +110,8 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
                             {RPCResult::Type::NUM, "version", "The peer version, such as 70001"},
                             {RPCResult::Type::STR, "subver", "The string version"},
                             {RPCResult::Type::BOOL, "inbound", "Inbound (true) or Outbound (false)"},
+                            {RPCResult::Type::BOOL, "bip152_highbandwidth_to", "Whether we selected peer as (compact blocks) high-bandwidth peer"},
+                            {RPCResult::Type::BOOL, "bip152_highbandwidth_from", "Whether peer selected us as (compact blocks) high-bandwidth peer"},
```

Comment:
The names have indeed become very long, and they should arguably be written as `bip152_high_bandwidth_{to/from}`, so I'd now vote for `bip152_hb_{to/from}`.

```
    "version" : n,                               (numeric) The peer version, such as 70001
    "subver" : "str",                            (string) The string version
    "inbound" : true|false,                      (boolean) Inbound (true) or Outbound (false)
    "bip152_highbandwidth_to" : true|false,      (boolean) Whether we selected peer as (compact blocks) high-bandwidth peer
    "bip152_highbandwidth_from" : true|false,    (boolean) Whether peer selected us as (compact blocks) high-bandwidth peer
    "addnode" : true|false,                      (boolean) Whether connection was due to addnode/-connect or if it was an automatic/inbound connection
    "startingheight" : n,                        (numeric) The starting height (block) of the peer
```

Reply:
Done, see comment below.